src/handlers/expenditures/expenditurePayoutClaimed.ts

```diff
@@ -25,6 +27,7 @@ export default async (event: ContractEvent): Promise<void> => {
     slot,
     token: tokenAddress,
     tokenPayout: amountWithoutFee,
+    agent: initiatorAddress,
```

Comment:
Just checking that you want this notification to be sent to the user who claimed the payout, and not the recipient/owner of the expenditure?

Reply:
It should just be sent colony wide, it's not a specific notification. The initiator address will be used to get information on who triggered the notification (creator), but in reality is isn't really used for anything other than debugging.